Orso.Arpa.Api/Startup.cs

```diff
@@ -108,6 +116,26 @@ public void ConfigureServices(IServiceCollection services)
             ConfigureAuthorization(services);
         }
 
+        protected virtual void ConfigureLocalization(IServiceCollection services)
+        {
+            if (services == null)
+                throw new ArgumentNullException(nameof (services));
+            var lz = new LocalizerCache(services);
+            services.AddSingleton(_ => lz);
+            services.AddSingleton<ArpaContext.CallBack<Translation>>(_ => lz.CallBack);
+            services.AddSingleton<IStringLocalizerFactory, ArpaLocalizerFactory>();
+
+            services.AddLocalization();
+
+            services.Configure<RequestLocalizationOptions>(options =>
+            {
+                options.SetDefaultCulture("en-GB");
+                options.AddSupportedUICultures("en", "en-GB", "de", "de-DE");
```

Comment:
Die unterstützten Sprachen sollte man vielleicht in die appsettings auslagern, so dass sie konfiguriert werden können, ohne das Backend neu kompilieren zu müssen

Reply:
Done